files/zh-cn/learn/html/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: 学习 HTML ：指南与教程
+slug: learn/HTML
+tags:
+  - HTML
+  - HTML5
+  - 主题
+  - 入门指导
+  - 基础
+  - 学习
+  - 新手
+  - 简介
+translation_of: Learn/HTML
+---
+{{LearnSidebar}}
+
+为了创建网站，你应该了解 {{Glossary('HTML')}} —— 用于定义一个网页结构的基本技术。HTML 用于表示你的网页内容是应该被理解为段落、列表、头部、链接、图像、多媒体播放器、表单或是其他众多可用的元素之一亦或是你定义的新元素。
+
+> **标注：**
+>
+> #### 希望成为一名前端网络开发人员？
+>
+> 我们已经所有你需要的放在一个课程里，
+> 以努力实现你的目标。
+>
+> [**开始**](/zh-CN/docs/Learn/Front-end_web_developer)
+
+## 须知
+
+在开始这个主题的学习之前，你至少要基本熟悉使用电脑和被动地使用网络（即单纯地查看内容）。你应该设置好一个基础的工作环境，详细参照 [安装基础软件](/zh-CN/docs/Learn/Getting_started_with_the_web/Installing_basic_software), 并且理解如何新建和管理文件，详细参照[文件处理](/zh-CN/docs/Learn/Getting_started_with_the_web/Dealing_with_files)——这两者都在 [Web 入门](/zh-CN/docs/Learn/Getting_started_with_the_web) 的完全新手模块里。
+
+在尝试学习这个主题之前，建议先完成 [Web 入门](/zh-CN/docs/Learn/Getting_started_with_the_web) 不过这并不是绝对必要的。[HTML 基础](/zh-CN/docs/Learn/Getting_started_with_the_web/HTML_basics)里大多数的文章在[HTML 介绍](/zh-CN/docs/Learn/HTML/Introduction_to_HTML)模块中也有涉及，不过要详细得多。。
```

Comment:
Remove the extra `。` in this end.

Reply:
update the document, using `git reset HEAD~1` and then commit with this changes. Or you can commit first and then using `git rebase` to merge the last two commit.

Reply:
merge it for now. you can add another pr to improve above point.